Examples/Python/python/acts/_adapter.py

```diff
@@ -84,9 +84,9 @@ def create(*args, mdecorator=None, **kwargs):
             cfg = config_class()
         _kwargs = {}
         for k, v in kwargs.items():
-            if hasattr(cfg, k):
+            try:
                 setattr(cfg, k, v)
-            else:
+            except AttributeError:
```

Comment:
Is `hasattr(cfg, k)` not covering some failure case or why the `try/catch`?

Reply:
Ah, this is because there's no getter, I see

Reply:
`hasttr()` tests `getattr()` works, not `setattr()`. In this case `jsonFile` declares the setter, not the getter, so wasn't picked up.

Reply:
Another way to work round this that I tried was to ctreate a dummy getter for `jsonFile`. I think this fix is neater as it solves the core problem.